HospitalSystem.Core/Equipment/Relocations/EquipmentRelocationService.cs

```diff
@@ -21,7 +21,7 @@ private void MoveEquipment(EquipmentRelocation relocation)
         var adding = new EquipmentBatch(relocation.ToRoomLocation, relocation.Name,
             relocation.Count, relocation.Type);
         _equipmentService.Remove(removing);
-        _equipmentService.Add(adding);
+        _equipmentService.Insert(adding);
         relocation.IsDone = true;
```

Comment:
This is not an Insert, this is an Add. It is intentionally different

Reply:
you told me to change all adds to inserts.. okay, will change it back